src/blog/2023/08/open-source-is-a-tier-not-competition.md

```diff
@@ -0,0 +1,70 @@
+---
+title: "Our Open Source offering is a tier, not our competition"
+subtitle: 
+description: Often our customers think we're competing against 
+date: 2023-08-18
+authors: ["zeger-jan-van-de-weg"]
+image: "/blog/2023/08/images/matt-hardy-waves.jpg"
+tags:
+    - posts
+    - flowfuse
+---
+
+More than once we’ve been in discussion with prospective customers on what tier
+is the right tier for their current Node-RED adoption. The question is likely to
+come up "Why wouldn't we just the open source version of FlowFuse?". The
+implicit discussion created is one that is alike the question:
+“Why wouldn’t we go with your competition?”. For FlowFuse, and most other open-core
+companies like us, the open licensed and free to use core is tier, not competition.
+
+<!--more-->
+
+In the traditional sense, the prospective customer is right. By the definition,
+a customer **buys** goods or services by exchanging it for **money**. For 
+FlowFuse’s open-source edition, which is free as in beer and free as in speech,
+no money changes hands. It can be installed an run by anyone. Once the software
+is being installed and used, For FlowFuse we consider we've gained a new customer.
+There’s an agreement in place, the Apache 2.0 license, and value is obtained by
+the customer. The only missing component compared to an ‘ordinary’ sale is the lack
+of money from the customer to FlowFuse. The fact that no monetary value is exchanged,
+like the situation where a customer picks the competitor, doesn’t make the open
+source tier competition. Just a free tier.
+
+Another reason it's really a tier is that the core of the product is the same. In
+many open-core products, the path to upgrade from the openly licenced product to
+the paid tier is much alike customers are used to on SaaS models. In the reals of
+self-managed software that's mostly uploading a license and  at times a few configuration
+steps.
+
+Furthermore, the open tier is a tier as the customer choses to not adopt all
+capabilities. They're leaving value on the table. Either this is because it's not
+quite clear what the value obtained is and if the price for the higher tiers provides
+enough business value to warrant the expense. Or the adoption journey for the
+customer doesn't require the fully featured tiers yet.
+The customer considers what they cannot adopt at the time, or indefinitely. In
```

Comment:
I don't understand this sentence?

Reply:
```suggestion
In
```

Trying to explain the decision is made anyway, there's no difference between buying a paid tier of OSS at this time. But it's only stressing it indeed.